config/GALE01/symbols.txt

```diff
@@ -26000,7 +26007,7 @@ lbl_804D5BF8 = .sdata:0x804D5BF8; // type:object size:0x8
 lbl_804D5C00 = .sdata:0x804D5C00; // type:object size:0x8
 lbl_804D5C08 = .sdata:0x804D5C08; // type:object size:0x8 data:4byte
 lbl_804D5C10 = .sdata:0x804D5C10; // type:object size:0x8 data:4byte
-lbl_804D5C18 = .sdata:0x804D5C18; // type:object size:0x8
+@32 = .sdata:0x804D5C18; // type:object size:0x2 scope:local data:string
```

Comment:
Does marking data as a string change it to `asciz` in the asm dump?

Reply:
I believe so, yes. This occurred automatically because of the code, not manual work that I did.